src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/provider/service/TaskManagementGetTaskByIdPactTest.java

```diff
@@ -0,0 +1,98 @@
+import au.com.dius.pact.provider.junitsupport.loader.VersionSelector;
+import au.com.dius.pact.provider.spring.junit5.MockMvcTestTarget;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Import;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.wataskmanagementapi.auth.access.AccessControlService;
+import uk.gov.hmcts.reform.wataskmanagementapi.controllers.TaskController;
+import uk.gov.hmcts.reform.wataskmanagementapi.domain.entities.task.Task;
+import uk.gov.hmcts.reform.wataskmanagementapi.services.CamundaService;
+
+import java.time.ZonedDateTime;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+
+@ExtendWith(SpringExtension.class)
+@Provider("wa_task_management_api_get_task_by_id")
+@PactBroker(scheme = "${PACT_BROKER_SCHEME:http}",
+    host = "${PACT_BROKER_URL:localhost}", port = "${PACT_BROKER_PORT:9292}", consumerVersionSelectors = {
+    @VersionSelector(tag = "master")})
+@Import(TaskManagementProviderTestConfiguration.class)
+@IgnoreNoPactsToVerify
+public class TaskManagementGetTaskByIdPactTest {
+
+    @Autowired
+    private AccessControlService accessControlService;
+
+    @Autowired
+    private CamundaService camundaService;
+
+    @TestTemplate
+    @ExtendWith(PactVerificationInvocationContextProvider.class)
+    void pactVerificationTestTemplate(PactVerificationContext context) {
+        if (context != null) {
+            context.verifyInteraction();
+        }
+    }
+
+    @BeforeEach
+    void beforeCreate(PactVerificationContext context) {
+        MockMvcTestTarget testTarget = new MockMvcTestTarget();
+        testTarget.setControllers(new TaskController(
+            camundaService,
+            accessControlService
+        ));
+        if (context != null) {
+            context.setTarget(testTarget);
+        }
+
+    }
+
+    @State({"appropriate task is returned"})
+    public void getTaskById() {
+        setInitiMock();
+    }
+
+    private void setInitiMock() {
+        when(camundaService.getTask(any(),any(),any())).thenReturn(createTask());
+    }
+
+    public Task createTask() {
+        return new Task("id",
+                         "Jake",
+                         "ReviewTheAppeal",
+                         "unconfigured",
+                         "taskSystem",
+                         "PRIVATE",
+                         "review",
+                         ZonedDateTime.now(),
+                         ZonedDateTime.now(),
+                         "TCW",
+                         true,
+                         "Time extension",
+                         "ia",
+                         "region",
+                         "12345",
+                         "Newcastle",
+                         "asylum",
```

Comment:
we should double check these values to make sure they are close to similar values e.g jurisdiction and caseType are keys so they should use the correct casing